be/src/agent/publish_version.cpp

```diff
@@ -163,6 +163,34 @@ void run_publish_version_task(ThreadPoolToken* token, const TPublishVersionReque
             pair.__set_version(task.max_continuous_version);
         }
     }
+    // return tablet and it's version which have already finished.
```

Comment:
It seems a little unclear to change the whole process in this way. The version will be set through both the task and the tablet. Can it be directly changed to set the version according to the tablet?

Reply:
Done